### PR TITLE
Added more unit tests

### PR DESCRIPTION
**Pull request by DyaryRaoof**
Added test for a function for editing the task description.
Added tests for a function for updating an item's 'completed' status.
Added unit tests for clear all completed function

### CODE REVIEW

File: __tests__/crud.test.js [hunk 1]
```diff
@@ -15,7 +15,10 @@ describe('add', () => {
 });
```
Comment (rloterh) [inline __tests__/crud.test.js:16]: - [ ] Test cases are failing. Kindly ensure to fix the same.

**Jest cases failing:**
image

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Dyary & Peter, 

You did extremely great on this project! 👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there! 🚦  🚦  

## Good Points 👍  
- No linter errors


## Required Changes ♻️
_Kindly check the comments under the review._

<br>
Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: __tests__/crud.test.js (hunk 1 above)
Comment (DyaryRaoof) [reply __tests__/crud.test.js:16]: Hi. it is passing all tests on my PC.  I set the `dom `environment to `jsdom `in `package.json.` So kindly use `npm test` to check for the tests thank you.

**Review by dasileker — APPROVED** [review]
### **Hi @DyaryRaoof,** 

### **Project Approved ✅**


Approved


Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀  
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**